Build failure on openjpeg-sys 1.0.4

### Simple Description on the Bug

The build has been failing on all of `ubuntu-latest`, `macOS-latest`, and `windows-latest` on GitHub Actions since [run on 2021-12-17 21:00 UTC](https://github.com/noritada/grib-rs/actions/runs/1594071217).

### Steps to Reproduce

1. Run `cargo build`

### Expected Behavior

Successful build is expected.

### Actual Behavior

Build fails with error messages like this (on ubuntu-latest):

```
error: linking with `cc` failed: exit status: 1
  |
  = note: "cc" "-m64" "/home/runner/work/grib-rs/grib-rs/target/debug/deps/gribber-80f0190eb821eb8f.1222uwsbdpw5s8e2.rcgu.o" (snip) "-Wl,--end-group" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-2a0b2a4f96acb821.rlib" "-Wl,-Bdynamic" "-lc" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc" "-Wl,--eh-frame-hdr" "-Wl,-znoexecstack" "-L" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-o" "/home/runner/work/grib-rs/grib-rs/target/debug/deps/gribber-80f0190eb821eb8f" "-Wl,--gc-sections" "-pie" "-Wl,-zrelro" "-Wl,-znow" "-nodefaultlibs"
  = note: /usr/bin/ld: /home/runner/work/grib-rs/grib-rs/target/debug/deps/libopenjpeg_sys-70c5f6d04fad6c88.rlib(t1.o): in function `opj_t1_clbl_decode_processor':
          /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/openjpeg-sys-1.0.4/vendor/src/lib/openjp2/t1.c:1690: undefined reference to `opj_t1_ht_decode_cblk'
          collect2: error: ld returned 1 exit status

  = help: some `extern` functions couldn't be found; some native libraries may need to be installed or have their path specified
  = note: use the `-l` flag to specify native libraries to link
  = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorustc-link-libkindname)
```

### Additional Context

Successful builds before [2021-12-16 21:00 UTC](https://github.com/noritada/grib-rs/actions/runs/1589417309) used openjpeg-sys 1.0.3, while builds after 2021-12-17 21:00 UTC uses 1.0.4, and this change may have
triggered the issue.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,7 +17,7 @@ clap = "2"
 console = "0.15"
 num = "0.4"
 num_enum = "0.5"
-openjpeg-sys = "1"
+openjpeg-sys = ">= 1.0.3, < 1.0.4"
 
 [target.'cfg(unix)'.dependencies]
 pager = "0.16"

